### ISSUE TITLE
icon-holder white background

### ISSUE THREAD

**Issue by yblock** [openstax/os-webview#2480]
The icon-holder has a white background set which results in an awkward drop shadow. On the new subjects page, looking at the icons used here and lower on the page for the subjects I notice that there are varying formats and transparencies used. Some are webp, some png, some have transparency and some do not. I believe this background should be removed and the images should be exported correctly (example for how this looks once fixed on the right) 

https://github.com/openstax/os-webview/blame/4ad6767a93d70698a7dec778cfa14904f62fc8b8/src/app/components/explore-by-subject/explore-by-subject.scss#L19

<img width="232" alt="Screenshot 2023-09-26 at 12 46 46 PM" src="https://github.com/openstax/os-webview/assets/5865292/3ca21f5b-d825-46c0-bba4-c1bc2543a8f0">
<img width="248" alt="Screenshot 2023-09-26 at 12 46 40 PM" src="https://github.com/openstax/os-webview/assets/5865292/e0138315-d23c-4578-8601-4e4d4155d6fa">

**Comment by RoyEJohnson** [comment]
If I change the background to transparent, it's differently not ideal (seeing the border through the icon). Is this what we want to go with? (It looks good with the business icon because it has a horizontal line in just the right place.)

![Image](https://github.com/openstax/os-webview/assets/7315241/157d82e6-08f4-4cab-8478-bb101344ace2)

**Comment by yblock** [comment]
I think this is correct, the art would just need some white fill added.

**Comment by erikayvonne** [comment]
Let’s pause on making this change. @andymuddimer should take a look - and I think we’ll need to be mindful about what we request from Micah due to capacity.

**Comment by yblock** [comment]
Just for context it takes about 10 seconds to use white bucket fill and save the image, of which there are only a few. (which is what I did when testing this and demoing this change, without any training in design even! haha)

**Comment by erikayvonne** [comment]
The issue is not the size of this individual request - I know this is very small - but when we're sending many small things folks' way, that ends up adding up. It's a bummer we're not able to work at full capacity, but that's where we're at. We'll be able to get to this eventually - I'd like to keep this in the backlog.

**Comment by micah-howard** [comment]
Hi! I'm happy to update these icons to include a white fill. I should be able to have all of these over by Friday. Is the below list everything? Are PNGs okay?

**Webinar (64x64px)**
- Business
- College Success
- Computer Science
- Humanities
- Math
- Nursing
- Science
- Social Sciences

**Blog (52x52px)**
- Business
- College Success
- Computer Science
- Humanities
- Math
- Nursing
- Science
- Social Sciences

**Comment by andymuddimer** [comment]
@micah-howard 

The files are stored in the Snippets for the Subjects, Blog Collections, and Webinar Collections
File types are any of AVIF, GIF, JPG, JPEG, PNG, WEBP. Maximum filesize: 2.0 MB.
PNG would give us the transparency - Yes
For sizes I believe the system converts the files to the correct size for each space
Please just work at your normal size and I will resize if needed

Subjects are all we need - just once - I can then upload to the Snippets

Please fill the "center" with white and leave the outside transparent
(let me know if you have questions)

**Comment by micah-howard** [comment]
@yblock @andymuddimer @erikayvonne Here are updated icons with a white fill: https://drive.google.com/drive/folders/1HQ_ZQFGZupzP1AB0kO0P5VQJuT9p46Gp?usp=sharing

**Comment by andymuddimer** [comment]
@yblock @RoyEJohnson  I have added all the new icons to the Subject Snippets on staging.. Can you try the change again to see if the box line looks correct now? Or do I need to do this on Dev?

**Comment by erikayvonne** [comment]
thanks so much @micah-howard !

**Comment by yblock** [comment]
Yes the images look good for this half of the work (and the images can be placed on prod without issue. The other half is whatever route @RoyEJohnson would like to take so there is not a white background added to the container. 

https://github.com/openstax/os-webview/assets/5865292/49fca7f5-3dce-416c-bb32-2707dca33163
